language/move-vm/runtime/src/load_metrics.rs

```diff
@@ -0,0 +1,97 @@
+use libra_metrics::{register_histogram, Histogram};
+use once_cell::sync::Lazy;
+
+//*******************Histogram Timers*********************//
+pub static MOVE_VM_LOAD_MODULE_EXPECT_NO_MISSING_DEPENDENCIES_SECONDS: Lazy<Histogram> = Lazy::new(|| {
+    register_histogram!(
+        // metric name
+        "MOVE_VM_LOAD_MODULE_EXPECT_NO_MISSING_DEPENDENCIES_SECONDS",
```

Comment:
These new metrics should follow the naming convention: "libra" + \<component-name\> + \<description\>. I suggest "movevm" for the component name here, and the names should all be lowercase. I also think you could skip the "metric name" and "metric description" comments here, but since it's a new file, I guess there's no existing convention to follow.